apps/docs/src/common/pages/get-started/project.tsx

```diff
@@ -0,0 +1,115 @@
+import { FC, Fragment, createElement as h } from 'react';
+import { PageProps } from '@not-govuk/app-composer';
+import { A } from '@not-govuk/components';
+
+
+
+const Page: FC<PageProps> = ({ location }) => (
+  <div className="govuk-grid-row">
+    <div className="govuk-grid-column-one-third">
+      <aside>
+        <h2>Get started</h2>
+          <ul className="plain">
+            <li><A href="/get-started/prototype">Prototyping for designers</A></li>
+            <li><A href="/get-started/project">Starting a project for developers</A></li>
+          </ul>
+      </aside>
+    </div>
+
+    <div className="govuk-grid-column-two-thirds">
+      <h1>
+        <span className="caption">Get started</span>
+        Starting a project for developers
+      </h1>
+        <p>This guide explains how to set up a project in production using the
+        coded React examples from the Home Office Design System.</p>
+
+        <p>In order to consume these components you will require a system that utilised a bundler (such as <A href="https://webpack.js.org/">Webpack</A>) that can process imported assets such as images, fonts and <A href="https://sass-lang.com/">Sass</A> files. <A href="https://create-react-app.dev/">Create React App</A> may be able to do this but does not provide Server-Side Rendering (SSR). You will also need to ensure that you provide an instance of <A href="https://reactrouter.com/">react-router</A>.</p>
+
+        <p>As such, we suggest that you use our specially designed tech stack for this purpose. You can start a brand new project or prototype using the following steps.</p>
+
+        <h2 className="govuk-heading-m">1. Set up your repository</h2>
+          <p>You should first set up a blank code repository in the provider of your choice. For the purposes of this document I will assume <A href="https://github.com/">GitHub</A> as we provide extra CI support through <A href="https://github.com/features/actions">GitHub Actions</A>.</p>
+
+        <h2 className="govuk-heading-m">2. Run the following commands locally</h2>
+          <p>The parts in ALL CAPS should be replaced with something specific to you:</p>
+          <ul>
+            <li>mkdir YOUR-PROJECT</li>
+            <li>cd YOUR-PROJECT</li>
+            <li>git init</li>
+            <li>git remote add origin git@github.com:YOUR_USER/YOUR-PROJECT.git</li>
+            <li>npm init @hods</li>
```

Comment:
```
npm init @hods project
```
(As we already know they want a project at this point.)

Reply:
Looks like we should add an extra line:
```shell
pnpm install
```
(We also might want to add a 'pre-requisites' section towards the top where we tell people they need Node.js installed and to install `pnpm` with:
```shell
npm install -g pnpm
```

Reply:
Added a 'Before you start' for both prototype and project sections. Is there a node version to include? Also we may need to think about whether the prototype section may need to include things like using Terminal for complete beginners..